### PR TITLE
Added customizations possibilities

### PR DESCRIPTION
**Pull request by Okipa**
Hi mmazzarolo,

According to the issue I have posted this morning (https://github.com/mmazzarolo/react-native-modal-datetime-picker/issues/3), here is a pull request that should let users do what they want now.

I didn't gave the possibility to show or not the cancel button. If you want to add this feature, that could be nice.

I'm just not sure about the PropTypes.number I set for the awaited Stylesheet objects for te props titleContainerStyleIOS, titleStyleIOS, ... It let you check this.

I also updated the readme.

Let me know if you find any problem.
Thanx for your work !

### CODE REVIEW

**Comment by mmazzarolo** [comment]
Man that's awesome! 
Thank you for your pull request, did you already test it? :)
Just one nitpick: can you set the styles proptypes to the more appropriate `View.propTypes.style`?

File: src/CustomDatePickerIOS/index.js [hunk 1]
```diff
@@ -14,12 +14,12 @@ export default class CustomDatePickerIOS extends Component {
         titleIOS: PropTypes.string,
         visible: PropTypes.bool,
         backgroundColorIOS: PropTypes.string,
-        titleContainerStyleIOS: PropTypes.number,
-        titleStyleIOS: PropTypes.number,
-        confirmButtonStyleIOS: PropTypes.number,
-        confirmButtonTextStyleIOS: PropTypes.number,
-        cancelButtonStyleIOS: PropTypes.number,
-        cancelButtonTextStyleIOS: PropTypes.number
+        titleContainerStyleIOS: PropTypes.style,
```
Comment (mmazzarolo) [inline src/CustomDatePickerIOS/index.js]: Ops, it should be `View.propTypes.style` 👍

**Comment by Okipa** [comment]
Ok, that should be good now.
I tested it and I found no problem.

**Comment by mmazzarolo** [comment]
Merged! I'll try to release this version on npm this evening 👍

**Comment by Okipa** [comment]
Nice, thanx.
Last improvement you could add (I just saw it) : adding the following styles in the **titleContainer** n CustomDatePickerIOS > index.style.js
```
borderTopLeftRadius: BORDER_RADIUS,
borderTopRightRadius: BORDER_RADIUS
```
If a background color is set on the title, its borders have currently no radius.

**Pull request merged**